### PR TITLE
Refactore Donates Modal

### PR DESCRIPTION
**Pull request by Lolimkeri**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/app/common/components/modals/Donates/DonatesModal.component.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import {
     useEffect, useRef, useState,
 } from 'react';
 
+import { Checkbox } from 'antd';
+
```
Comment (Katerix) [inline src/app/common/components/modals/Donates/DonatesModal.component.tsx:11]: empty line

File: src/app/common/components/modals/Donates/DonatesModal.styles.scss [hunk 2]
```diff
@@ -104,7 +113,39 @@
...
+        .ant-checkbox-inner {
+          width: 24px !important;
+          height: 24px !important;
+        }
+  
+        &:last-child {
+          font-family: 'Roboto';
+          font-style: normal;
+          font-weight: 300;
+          font-size: 15px;
```
Comment (Katerix) [inline src/app/common/components/modals/Donates/DonatesModal.styles.scss:141]: you can try using font-shorthand css (nice example is in w3)

**Review by Katerix — APPROVED** [review]
nice!

File: src/app/common/components/modals/Donates/DonatesModal.styles.scss (hunk 2 above)
Comment (Lolimkeri) [reply src/app/common/components/modals/Donates/DonatesModal.styles.scss:141]: Ok, will try it

**Pull request merged**